tee-worker/ts-tests/integration-tests/common/utils/assertion.ts

```diff
@@ -261,8 +278,132 @@ export async function checkJson(vc: any, proofJson: any): Promise<boolean> {
     expect(isValid).to.be.true;
     expect(
         vc.type[0] === 'VerifiableCredential' &&
-        vc.issuer.id === proofJson.verificationMethod &&
-        proofJson.type === 'Ed25519Signature2020'
+            vc.issuer.id === proofJson.verificationMethod &&
+            proofJson.type === 'Ed25519Signature2020'
     ).to.be.true;
     return true;
 }
+
+/* 
+    assert linked event
+
+    steps:
+    1. compare event account with signer
+    2. compare event identity with expected identity
+    3. compare event prime identity with expected prime identity
+    4. compare event web3networks with expected web3networks
+    5. check event idGraph LitentryPrimitivesIdentityContext(linkBlock > 0, isActive = true)
+*/
+
+export async function assertLinkedEvent(
+    context: IntegrationTestContext,
+    signer: KeyringPair,
+    events: any[],
+    expectedIdentities: LitentryPrimitivesIdentity[]
+) {
+    assert.isAtLeast(events.length, 1, 'Check assertLinkedEvent error: events length should be greater than 1');
+
+    const eventIdGraph = parseIdGraph(context.sidechainRegistry, events[events.length - 1].data.idGraph, aesKey);
+
+    const keyringType = signer.type;
+    for (let index = 0; index < events.length; index++) {
+        const eventData = events[index].data;
+        // evm address should convert to substrate address before compare
+        const who = keyringType === 'ethereum' ? eventData.account.toHuman() : eventData.account.toHex();
+        const signerEvmToSubstrateAddress =
+            keyringType === 'ethereum'
+                ? polkadotCryptoUtils.evmToAddress(signer.address, context.chainIdentifier)
+                : u8aToHex(signer.addressRaw);
+
+        // step 1
+        assert.equal(
+            who,
+            signerEvmToSubstrateAddress,
+            'Check IdentityLinked error: signer should be equal to who, but got ' +
+                who +
+                ' and ' +
+                signerEvmToSubstrateAddress
+        );
+
+        // step 2
+        // parse event identity
+        const eventIdentity = parseIdentity(context.sidechainRegistry, eventData.identity, aesKey);
+        // prepare expected identity
+        const expectedIdentity = expectedIdentities[index];
+        // compare identity
+        assert.equal(
+            eventIdentity.toHuman()?.toString(),
+            expectedIdentity.toHuman()?.toString(),
```

Comment:
This can be undefined? :raised_eyebrow:

Reply:
The reason is that `toHuman()` might return undefined. It seems that t`oHuman()` is unnecessary, and using `toString()` directly should be sufficient.
<img width="545" alt="image" src="https://github.com/litentry/litentry-parachain/assets/104152026/7d58d1dc-84cd-40d6-b898-0227b273afc7">